packages/turbo-ignore/src/types.ts

```diff
@@ -1,3 +1,15 @@
+export type NonFatalErrorKeys =
```

Comment:
This is a singular type, e.g. `NonFatalErrorKey`

Reply:
Good call, will fix